Src/Newtonsoft.Json.Tests/Newtonsoft.Json.Tests.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(TestFrameworks)'==''">net46;net40;net35;net20;net5.0;netcoreapp3.1;netcoreapp2.1</TargetFrameworks>
+    <!--TargetFrameworks Condition="'$(TestFrameworks)'==''">net6.0;net46;net40;net35;net20;net5.0;netcoreapp3.1;netcoreapp2.1</TargetFrameworks-->
+    <TargetFrameworks Condition="'$(TestFrameworks)'==''">net6.0;net5.0;netcoreapp3.1;netcoreapp2.1</TargetFrameworks>
```

Comment:
These will of course need to be restored before merging, I didn't want to install all the targeting packs all the way back to net2.0 though.